src/decisionengine/framework/modules/logging_configDict.py

```diff
@@ -83,7 +84,7 @@
         structlog.stdlib.PositionalArgumentsFormatter(),
         structlog.processors.StackInfoRenderer(),
         structlog.processors.format_exc_info,
-        structlog.processors.JSONRenderer(sort_keys=True),
+        structlog.processors.JSONRenderer(sort_keys=False),
```

Comment:
Any special reason to disable sorting?

Reply:
I was playing with the ordering of the output dict items in the structured log. With the sorting set to `True` the ordering is alphabetical:
`{"channel": "engine", "event": "de logging setup complete", "level": "debug", "logger": "decisionengine", "module": "de_logger", "timestamp": "2021-08-31 17:23:44"}
`
with it set to `False` the output ordering is this:
`{"module": "de_logger", "channel": "engine", "event": "de logging setup complete", "logger": "decisionengine", "level": "debug", "timestamp": "2021-08-31 17:18:53"}`

I preferred the latter ordering for debugging so I switched it.

Reply:
Agree